corehq/apps/geospatial/utils.py

```diff
@@ -221,8 +221,9 @@ class CeleryTaskTracker(object):
     Simple Helper class using redis to track if a celery task was requested and is not completed yet.
     """
 
-    def __init__(self, task_key):
+    def __init__(self, task_key, message_key=None):
```

Comment:
I implemented it as such thinking it might be nice to have exact control over the defined keys for the task tracker. But you make a good point, and there's no inherent benefit to keeping them separate. Another benefit of doing this would be that we can completely remove the helper function since we can just call the `CeleryTaskHelper` class directly with a single task key.

Reply:
Addressed in [196ffd3](https://github.com/dimagi/commcare-hq/pull/35126/commits/196ffd3fd025e3a8a5999498ef91eae399c34225).